doc/classes/PhysicsDirectSpaceState.xml

```diff
@@ -17,6 +19,8 @@
 			<argument index="1" name="motion" type="Vector3">
 			</argument>
 			<description>
+				Check whether the shape can travel to a point. If it can, the method will return an array with two floats: The first is the distance the shape can move in that direction without colliding, and the second is the distance at which it will collide.
```

Comment:
I'm still wondering about the different return values.  They both sound like the distance to the collision to me?

Reply:
@brainsick One will be *slightly* larger than the other. How much larger depends on how often the physics engine checks for collisions, I think.